services/custom_rule.go

```diff
@@ -0,0 +1,53 @@
+package services
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/muety/wakapi/models"
+)
+
+type CustomRuleService struct {
+	Config *models.Config
+	Db     *gorm.DB
+}
+
+func NewCustomRuleService(db *gorm.DB) *CustomRuleService {
+	return &CustomRuleService{
+		Config: models.GetConfig(),
+		Db:     db,
+	}
+}
+
+func (srv *CustomRuleService) GetCustomRuleById(CustomRuleId uint) (*models.CustomRule, error) {
```

Comment:
Minor: please make parameter names lower case.

Reply:
Fixed it